src/CoreWCF.NetNamedPipe/src/CoreWCF/Channels/NamedPipeListener.cs

```diff
@@ -40,6 +40,7 @@ public NamedPipeListener(NamedPipeListenOptions options)
             _logger = options.ApplicationServices.GetRequiredService<ILogger<NamedPipeListener>>();
             _framingConnectionLogger = options.ApplicationServices.GetService<ILogger<FramingConnection>>();
             _transportConnectionManager = new TransportConnectionManager();
+            AddNamedPipeListenOptionsToConnectionContext(options);
```

Comment:
Does this relate to the dependencies update?

Reply:
It doesn't. One of the PR runs triggered this bug to manifest. We've had a couple of new bugs come up with .NET 9 having different timing and this was one of them.